config/scorecard/patches/basic.config.yaml

```diff
@@ -4,7 +4,7 @@
     entrypoint:
     - scorecard-test
     - basic-check-spec
-    image: quay.io/operator-framework/scorecard-test:v1.1.0
+    image: icr.io/operator-framework/scorecard-test:v1.1.0
```

Comment:
This line likely should not be updated (this is not one of our owned images that we are moving, and probably does not exist in icr)